_includes/footer.html

```diff
@@ -1,100 +1,26 @@
-        </div>
-
-        <footer class="footer mt-60">
-            <section id="social-media" class="container pt-60">
-                <div class="row">
-                    <div class="col-xs-12 text-center">
-                        <ul class="social white">
-                            <li class="link mr-0" style="padding: 0 10px">
-                                <a class="white" href="https://groups.google.com/forum/#!forum/gaffer-dev"><i class="fa fa-google-plus-square" style="font-size: 36px"></i></a>
-                            </li>
-                            <li class="link mr-0" style="padding: 0 10px">
-                                <a class="white" href="https://github.com/gafferHQ/gaffer" style="font-size: 36px"><i class="fa fa-github-square"></i></a>
-                            </li>
-                            <li class="link mr-0" style="padding: 0 10px">
-                                <a class="white" href="https://vimeo.com/cortex" style="font-size: 36px"><i class="fa fa-vimeo-square"></i></a>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </section>
-            <section id="copyright" class="container copyright ptb-60">
-                <div class="row">
-                    <div class="col-xs-12">
-                        <p class="text-uppercase lead white"><a href="{{ site.baseurl }}" class="white">Gaffer</a></p>
-                        <p class="white">© {{ site.copyrightYear }} Image Engine Design Inc. All rights reserved.<br></p>
-                        <p class="white">© {{ site.copyrightYear }} John Haddon. All rights reserved.</p>
-                    </div>
-                </div>
+<footer class="mt-5 pt-4 pb-2 bg-dark gaff-text-gray-400 small" id="footer">
+    <div class="container">
+        <div class="row text-center text-md-left">
+            <section class="col-12 col-md-6 mb-3 mb-md-0" id="sitemap">
+                <ul class="list-unstyled">
+                    <li class="mb-1"><a href="#top">Top</a></li>
+                    <li class="mb-1"><a href="/download">Home</a></li>
+                    <li class="mb-1"><a href="/download">Download</a></li>
+                    <li class="mb-1"><a href="/news">News</a></li>
+                    <li class="mb-1"><a href="https://gafferhq.org/documentation/{{ site.latest-release }}">Docs</a></li>
```

Comment:
Can we add `www` to the docs URL? otherwise you get HTTPS errors and the 'this site is evil and will steal your cat' interstitial UX.

`NET::ERR_CERT_COMMON_NAME_INVALID`